### ISSUE TITLE
[Xiaomi Mi A2] Fingerprint doesn't work

### ISSUE THREAD

**Issue by cuesta4** [phhusson/treble_experimentations#463]
I've flashed v112 with 8.1 and 9.0 stock vendors, but none of them work. Might be related to the Enforcing SE Linux, but I don't know. 

If you need any logs, how can I provide them? Thanks in advance.

**Comment by suwakowww** [comment]
Did you have **Goodix** fingerprint?  Check with following command:
````
getprop persist.sys.fp.vendor
getprop ro.boot.fpsensor
getprop sys.fp.vendor
````
It seems similar to Mi 6X (#237), but I don't know how to fix it because I got **FPC** fingerprint, and it's working well.

From @TadiT7's ROM dumps, you can see [jasmine](https://github.com/TadiT7/xiaomi_jasmine_dump/tree/jasmine-user-9-PKQ1.180904.001-V10.0.1.0.PDIMIFJ-release-keys/system/system/etc/firmware), [wayne](https://github.com/TadiT7/xiaomi_wayne_dump/tree/wayne-user-9-PKQ1.180904.001-9.3.11-release-keys/system/etc/firmware), [whyred](https://github.com/TadiT7/xiaomi_whyred_dump/tree/whyred-user-9-PKQ1.180904.001-9.3.25-release-keys/system/etc/firmware), [tulip](https://github.com/TadiT7/xiaomi_tulip_dump/tree/tulip-user-9-PKQ1.180904.001-9.3.11-release-keys/system/etc/firmware), [lavender](https://github.com/TadiT7/xiaomi_lavender_dump/tree/lavender-user-9-PKQ1.180904.001-V10.2.10.0.PFGCNXM-release-keys/system/system/etc/firmware) and [violet](https://github.com/TadiT7/xiaomi_violet_dump/tree/violet-user-9-PKQ1.181203.001-V10.2.6.0.PFHINXM-release-keys/system/system/etc/firmware) are similar, they all put some files for fingerprint into `/system/etc/firmware` or `/system/system/etc/firmware`

**Comment by cuesta4** [comment]
Yeah, it does have Goodix fingerprint. When browsing through /system, I've seen some apps related to the fingerprint scanner.

**Comment by phhusson** [comment]
Could you try copying /system/etc/firmware folder from stock rom?
